src/checked.js

```diff
@@ -0,0 +1,13 @@
+const checkedEvent = (event, toDoTask) => {
+  if (event.target.classList.contains('check-box')) {
+    if (event.target.checked) {
+      const taskToBeFlaged = toDoTask.tasksToDo.find(
+        (task) => task.index === Number(event.target.parentNode.id),
+      );
+
+      taskToBeFlaged.completed = true;
+    }
+  }
+};
+
```

Comment:
- [ ] It would be better if you update the status of the task when you clicked on the check box 

![tempsnip](https://user-images.githubusercontent.com/64173189/154574761-fd5285a0-233c-415f-806d-6a4f313efaf3.png)

Kindly fix this.

- [ ] Your app is working with some bugs. Please fix these bugs. 💪 😄 


https://user-images.githubusercontent.com/64173189/154574858-b70eb16c-a6fa-4870-ac5a-0bfcdb3b7759.mp4